ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/metering_report_service.proto

```diff
@@ -50,6 +50,9 @@ message GetMeteringReportResponse {
 
     // The time at which the report was computed.
     google.protobuf.Timestamp report_generation_time = 3;
+
+    // The metering report json
+    string metering_report_json = 4;
```

Comment:
```suggestion
    google.protobuf.Struct metering_report_json = 4;
```
Use https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Struct as per https://groups.google.com/g/protobuf/c/1772tpAr3N4

Reply:
This allows tools like `grpcurl` to properly render the JSON data.

Reply:
Yes it does.

# With string
```
{
  "request": {
    "from": "2020-01-01T00:00:00Z"
  },
  "participant_report": {
    "participant_id": "default"
  },
  "report_generation_time": "2022-07-21T17:33:38.579640Z",
  "metering_report_json": "{\"participantId\":\"default\"}"
}
```

# With struct
```
{
  "request": {
    "from": "2020-01-01T00:00:00Z"
  },
  "participant_report": {
    "participant_id": "default"
  },
  "report_generation_time": "2022-07-22T08:40:03.955245Z",
  "metering_report_json": {
      "participantId": "default"
    }
}
```

Reply:
Looks good. This can now be resolved can it?

Reply:
Yes, the about was output from `grpcurl`